app/(with-main-header)/albatalk/[talkId]/_components/CommentItem.tsx

```diff
@@ -0,0 +1,49 @@
+'use client';
+import Image from 'next/image';
+import EditDropdown from './EditDropdown';
+import { formatDate } from '@/utils/dateFormatter';
+import { EditDropdownAction } from '@/types/albatalk';
+import { Comment } from '@/types/albatalk';
+
+interface CommentItemProps {
+  comment: Comment;
+  userId: number | null;
+  onAction: (action: EditDropdownAction) => void;
+}
+
+const CommentItem = ({ comment, userId, onAction }: CommentItemProps) => {
+  return (
+    <div className="flex flex-col gap-4">
+      <div className="flex flex-col gap-6">
+        <div className="flex w-full justify-between items-center">
+          <div className="flex gap-4 items-center">
+            <div className="flex gap-2 items-center">
+              <div className="relative w-4 h-4 lg:w-6 lg:h-6">
+                <Image
+                  src={comment.writer.imageUrl || '/icons/profile.svg'}
+                  alt="user profile"
```

Comment:
💊  alt 문자열이 항상 고정되어있는데
```suggestion
                  alt={`${comment.writer.nickname}'s profile`}
```
작성자 닉네임을 넣으면 접근성이 더 향상 될것같아요